src/pages/Admin/Admin.tsx

```diff
@@ -456,24 +455,22 @@ const Admin: React.FC = () => {
   }
 
   return (
-    <WorldUsersProvider venueId={venueId}>
```

Comment:
[note] Removed from here because we provide it higher up in `src/components/organisms/AppRouter/AdminSubrouter.tsx` now.